eng/mgmt/automation/generation.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  NodeVersion: '14.x'
+
+steps:
+- task: Cache@2
+  inputs:
+    key: 'maven | "$(Agent.OS)" | **/pom.xml'
+    restoreKeys: |
+      maven | "$(Agent.OS)"
+      maven
+    path: $(MAVEN_CACHE_FOLDER)
+  displayName: Cache Maven local repo
+
+- bash: |
+    git config --global user.name "Azure-Fluent"
+    git config --global user.email "azfluent@microsoft.com"
+  displayName: Base Config
+
+- bash: |
+    sudo apt-get install -y --upgrade python3-pip python3-setuptools
+    pip3 install --upgrade wheel
+    pip3 install --upgrade PyYAML
+  displayName: Update python
+
+- task: NodeTool@0
+  displayName: 'Install Node.js $(NodeVersion)'
+  inputs:
+    versionSpec: '$(NodeVersion)'
+
+- bash: |
+    npm install -g autorest
+    npm install
+  displayName: 'Install autorest'
+
+- bash: |
+    export PATH=$JAVA_HOME_11_X64/bin:$PATH
+    java -version
+    ./eng/mgmt/automation/generate.py --readme "$(README)" --tag "$(TAG)" --service "$(SERVICE)" --version "$(VERSION)" --auto-commit-generated-code
+  displayName: Generation and Compile
+
+- bash: |
+    export MESSAGE="[Automation] Generate Fluent Lite from $(README)#$(TAG)"
+    export BRANCH="Fluent-lite-generation-$(Build.BuildId)"
+    git add -A
+    git commit -m "[Automation] Update other files outside generation folder"
+    git log -2
+    git push https://$(GITHUB_TOKEN)@github.com/azure-fluent/azure-sdk-for-java HEAD:refs/heads/$BRANCH -f
+    python3 eng/mgmt/pull-request.py chentanyi azure-sdk-for-java "$MESSAGE" "azure-fluent:$BRANCH"
```

Comment:
orthogonal but it would be good for use to use the shared code to submit pull requests, see https://github.com/Azure/azure-sdk-for-java/blob/master/eng/common/pipelines/templates/steps/create-pull-request.yml.

Reply:
As the [yml show](https://github.com/azure/azure-sdk-for-java/blob/master/eng/common/pipelines/templates/steps/create-pull-request.yml#L1), it need the `github-pat` of `azure-sdk`, is the pat will auto-apply to all pipeline in azure-sdk? Or I would use `Azure-Fluent` account instead?

Reply:
Unless there is a reason you need to use the azure-fluent account specifically I'd suggest just using the azure-sdk service account like a lot of our other automation does.

Reply:
Yes, I think we could use the `azure-sdk` account. So I would like to know do we need to set the `azure-sdk` github-pat manually in the pipeline?

Reply:
change to azure-sdk done

Reply:
You don't set it manually in the pipeline but instead hook up the correct variable group that has the secret which is pulled from keyvault. I